src/theme/includes/webfonts.html

```diff
@@ -0,0 +1,6 @@
+<script>
+  {% include 'assets/scripts/loadCss.js' %}
+</script>
+<script>
+  loadCSS('https://fonts.googleapis.com/css?family=Open+Sans:400,700');
```

Comment:
Keeping things simple for now in the default theme. Google recommends using the embed code /and serves `woff` / `woff2` depending on the browser used.